hw/top_earlgrey/formal/top_earlgrey_fpv_cfgs.hjson

```diff
@@ -53,6 +53,14 @@
                rel_path: "hw/ip/prim/prim_arbiter_fix/{sub_flow}/{tool}"
                cov: true
              }
+             {
+               name: prim_count_fpv
+               fusesoc_core: lowrisc:fpv:prim_count_fpv
```

Comment:
Unlike DV, FPV bench can instantiate multiple testbenchs, like LFSR: https://github.com/lowRISC/opentitan/blob/master/hw/ip/prim/fpv/tb/prim_lfsr_fpv.sv
I think this method is more preferred because it can avoid generating multiple hjson file.

Reply:
I will suggest rethinking this strategy. You are making JG work harder to converge on multiple instances in the same run. Instead, if you make them separate runs, you can leverage parallelism. We should put the vast GCP resources to good use :)

Reply:
Also to make this worse, you already have 4 instances in this testbench - you will need to cross all widths with these 4 flavors, which means 4X the number of widths as instances.

Reply:
I see your points! That makes sense. I will separate them and create one entry for each. Thanks for explaining!